packages/amplify-category-analytics/src/index.ts

```diff
@@ -1,7 +1,6 @@
 import { $TSContext, $TSAny, AmplifyFault } from 'amplify-cli-core';
 import * as path from 'path';
-import inquirer, { QuestionCollection } from 'inquirer';
```

Comment:
don't see changes in package.json, we need to remove the references for inquirer from dependencies section as well

Reply:
Yep, will do that now.